src/Sulu/Bundle/ContentBundle/Content/Structure/ExcerptStructureExtension.php

```diff
@@ -90,7 +90,10 @@ public function __construct(
      */
     public function save(NodeInterface $node, $data, $webspaceKey, $languageCode)
     {
-        foreach ($this->excerptStructure->getProperties() as $property) {
+        $excerptStructure = $this->getExcerptStructure();
```

Comment:
I doubt that it make sense to cache this class, if the value has always to be adapted... Why is it done this way?